main.py

```diff
@@ -73,12 +76,15 @@ def set_arg_parser():
     stop_framework = False
     while not stop_framework:
         try:
+            1/0
```

Comment:
maybe remove that. It's a feature and we are in feature lock :(

Reply:
Its my favorite feature :(